src/components/Login/Login.tsx

```diff
@@ -131,22 +130,19 @@ export default class LoginModal extends Component<Partial<LoginModalProps>, Logi
 
   public submitCredentials = async () => {
     const { form } = this.state;
-    const username = (form.username && form.username.value) || '';
-    const password = (form.password && form.password.value) || '';
+    const username = form.username?.value ?? '';
```

Comment:
null coalescing operator and optional chaining at work 💪

Reply:
nice !! glad you can use it already !

Reply:
Do we transpile this? It is only availble in Node 13+ afaik.